packages/ui-components/src/chat/chat.tsx

```diff
@@ -0,0 +1,228 @@
+/**
+ * Copyright 2023 Impulse Innovations Limited
+ *
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as React from 'react';
+import { v4 as uuidv4 } from 'uuid';
```

Comment:
We already have two libraries for generating random IDs: `shortid` which is one we're trying to get rid of and is deprecated, and `nanoid` which should be used instead. Let's use the latter rather than bringing a third one please 😅

Reply:
Fair, forgot about those, updated